scripts/parse_tools/parse_checkers.py

```diff
@@ -221,6 +222,36 @@ def check_fortran_id(test_val, prop_dict, error, max_len=0):
 
 ########################################################################
 
+def fortran_list_match(test_str):
+    """Check if <test_str> could be a list of Fortran expressions.
+    The list must be enclosed in parentheses and separated by commas.
```

Comment:
Ahh, my good old school education ... the real plural of comma is commata, but of course nowadays commas is ok (just like these insane minimums and all that stuff - it's minima!). Getting side-tracked.

Reply:
Old school or Old World? While it is an accepted plural (imported from the Greek), my Merriam Webster (10th edition) has commas as the default plural.